cloud/auth/auth.go

```diff
@@ -356,7 +357,7 @@ func checkToken(c *config.Context, client astro.Client, out io.Writer) error {
 }
 
 // Login handles authentication to astronomer api and registry
-func Login(domain string, client astro.Client, out io.Writer, loginLink bool) error {
+func Login(domain string, client astro.Client, out io.Writer, loginLink, tokenLogin bool) error {
```

Comment:
I think tokenLogin should be renamed to `shouldLoginWithToken` for more clarity on what the behaviour is

Reply:
updated this